src/go/rpk/pkg/plugin/plugin.go

```diff
@@ -77,9 +77,26 @@ func NameToArgs(command string) []string {
 	return strings.Split(command, "_")
 }
 
-// UserPaths returns the user PATH list.
+// GetDefaultBinPath returns HOME-DIR/.local/bin where HOME-DIR is the directory
+// returned by os.UserHomeDir.
+func GetDefaultBinPath() (string, error) {
```

Comment:
I think this can be a private function right now

Reply:
We use it in `install.go`:
```go
func determineBinDir(fs afero.Fs) (string, error) {
	path, err := plugin.GetDefaultBinPath()
	if err != nil {
		return "", err
	}

	if exists, _ := afero.DirExists(fs, path); !exists {
	...
```

https://github.com/redpanda-data/redpanda/pull/6909/files#diff-a233cf26e268f01629bda87ab9ad758694a35a37b17497b283cf61428a6a59c6R142